package.json

```diff
@@ -25,8 +25,10 @@
     "electron": "^25.1.1"
   },
   "dependencies": {
+    "axios": "^1.4.0",
     "better-sqlite3": "^8.4.0",
     "bootstrap": "^5.3.0",
+    "electron-dl": "^3.5.0",
```

Comment:
are these new dependencies needed?

Reply:
We need the electron-dl one but we don't need axios I'll remove that one